tsconfig.production.json

```diff
@@ -3,8 +3,8 @@
     "allowSyntheticDefaultImports": true,
     "alwaysStrict": true,
     "charset": "utf-8",
-    "declaration": true,
-    "declarationMap": true,
+    "declaration": false,
+    "declarationMap": false,
```

Comment:
Plugins can be consumed as regular JavaScript packages. The same way as this plugin is developed in TypeScript, other plugins which consume it might do the same. We should still ship declaration files for these use cases.

Reply:
I don't know of any benefit of using an ESlint plugin directly. Therefore the declarations are really meaningless.
This is not uncommon in ESlint plugins built with TypeScript.
See [typescript-eslint](https://github.com/typescript-eslint/typescript-eslint/blob/176054c2171b682217d6855208e50b15e1712675/packages/eslint-plugin/tsconfig.build.json#L5-L6)
See [eslint-plugin-jest](https://github.com/jest-community/eslint-plugin-jest/blob/master/tsconfig.json)

We can ultimately add them but to my view it amounts to nothing, if I am missing something please let me know 😄